qmail-getpw.c

```diff
@@ -47,6 +47,7 @@ int userext()
 	    }
 	    else
 	      if (error_temp(errno)) _exit(QLX_NFS);
+          }
```

Comment:
The indentation here looks wrong.

Reply:
DJB's indentation was a mix of spaces and what seem to have been 8-space tabs. Check locally in your editor with brace-matching and it should look (somewhat) better.

When we eventually defeat patching, globally replacing tabs with spaces will give me a great deal of satisfaction.

Reply:
Should be "Tab, Space, Space" before the brace then to match the if.

Reply:
Wow. Eagle eyes.